sciezka.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+from numpy import inf
+import math
+
+class Road:
+    def __init__(self, lidar_map : np.array, starting_point : tuple, ending_point : tuple, resolution = 1.0, car_length = 1.5, car_width = 0.5):
```

Comment:
Domergujcie sobie branch `LidarScan-to-LidarMap` i wykorzystajcie klase mapy od chłopaków